packages/utilities/fast-web-utilities/src/array.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Returns the index of the last element in the array where predicate is true, and -1 otherwise.
+ *
+ * @param array - the array to test
+ * @param predicate - find calls predicate once for each element of the array, in descending order, until it finds one where predicate returns true. If such an element is found, findLastIndex immediately returns that element index. Otherwise, findIndex returns -1.
+ */
+export function findLastIndex<T>(
+    array: Array<T>,
+    predicate: (value: T, index: number, obj: T[]) => unknown
```

Comment:
```suggestion
    predicate: (value: T, index?: number, obj?: T[]) => unknown
```

Reply:
This matches the signature of the built-in `Array.prototype.findIndex` function.

Reply:
I flipped back and forth on making the params optional, but after testing I realized they need to be there to avoid errors when `strictNullChecks` is enabled in #5378